### PR TITLE
Add version_check callback plugin.

### PR DESCRIPTION
**Pull request by tjanez**
This callback module checks if the Ansible version that is used to run
the playbook matches the Ansible version specified in the requirements
file.

### CODE REVIEW

File: lib/ansible/plugins/callback/version_check.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    If the version of Ansible is not specified, return None.
+
+    """
+    version = None
+    with open(req_path) as req_file:
+        for req in req_file.readlines():
+            req = req.strip()
+            if req.lower().startswith("ansible"):
+                req_split = req.split("==")
```
Comment (ueg1990) [inline lib/ansible/plugins/callback/version_check.py:43]: you should also include the separator '>='

File: lib/ansible/plugins/callback/version_check.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+class CallbackModule(CallbackBase):
+    """
+    This callback module checks if the Ansible version that is used to run the
+    playbook matches the Ansible version specified in the requirements file.
+
+    To use it, just place 'requirements.txt' file in your playbook directory
+    with the following contents (replace '2.0.0' with your required version):
+
+    ansible=2.0.0
```
Comment (ueg1990) [inline lib/ansible/plugins/callback/version_check.py:58]: ansible==2.0.0

**Comment by bcoca** [comment]
I'm not sure that a callback is where this belongs, we might be better off by adding a version pragma or just have a task checking the ansible_version variable.

**Comment by sivel** [comment]
Question About This One
=======================

Hi!

Thanks very much for your submission to Ansible.  It sincerely means a lot to us that you've taken time to contribute.

Unfortunately, we're not sure if we want this feature in the program, and I don't want this to seem confrontational.  

However, we're absolutely always up for discussion.  Since this is a really busy project, we don't always see comments on closed tickets, but want to encourage
open dialog.  You can stop by the development list, and we'd be glad to talk about it - and we might even be persuaded otherwise!

   * https://groups.google.com/forum/#!forum/ansible-devel

In the future, sometimes starting a discussion on the development list prior to implementing a feature can make getting things included a little easier, but it's not always necessary.

Thank you once again for this and your interest in Ansible!

**Pull request closed without merging**